utils/Framework/Node/OakNode.ts

```diff
@@ -0,0 +1,120 @@
+import { ApiPromise, WsProvider } from "@polkadot/api";
+import { Extrinsic } from "../../setup";
+
+export class OakApi {
+  api: ApiPromise;
+
+  addChainCurrencyData(paraId: number, currencyId: number): Extrinsic {
+    return this.api.tx.xcmpHandler.addChainCurrencyData(paraId, currencyId, {
+      native: false,
+      feePerSecond: 537_600_000_000,
+      instructionWeight: 150_000_000 * 6,
+    });
+  }
+
+  constructor(api: ApiPromise) {
+    this.api = api;
+  }
+
+  static async create(uri: string): Promise<OakApi> {
```

Comment:
You shouldn’t need to create those definitions yourselt. Importing oak.js should do.
```
import '@oak-foundation/api-augment'
import { rpc } from '@oak-foundation/types';
import { ApiPromise, WsProvider } from '@polkadot/api'
```


Please check out this quick demo, https://docs.oak.tech/docs/oak-js/
Oak.js Code, https://github.com/OAK-Foundation/oak.js